iocore/net/UnixNet.cc

```diff
@@ -96,8 +97,8 @@ class InactivityCop : public Continuation
           NET_SUM_DYN_STAT(keep_alive_queue_timeout_total_stat, diff);
           NET_INCREMENT_DYN_STAT(keep_alive_queue_timeout_count_stat);
         }
-        Debug("inactivity_cop_verbose", "vc: %p now: %" PRId64 " timeout at: %" PRId64 " timeout in: %" PRId64, vc, now,
-              vc->next_inactivity_timeout_at, vc->inactivity_timeout_in);
+        Debug("inactivity_cop_verbose_2", "vc: %p now: %" PRId64 " timeout at: %" PRId64 " timeout in: %" PRId64, vc,
```

Comment:
I just realized that in the patch I pasted in TS-3922 I left this debug tag name changed as inactivity_cop_verbose_2, @zizhong can you please correct that for me.